src/main/java/net/revelc/apilyzer/maven/plugin/Analyze.java

```diff
@@ -123,25 +117,17 @@ private boolean checkClass(Class<?> clazz, Set<String> publicSet, PrintStream ou
       return true;
     }
 
-    Class<?> superClazz = clazz.getSuperclass();
-    if (superClazz != null) {
-      ok = checkSuperClass(clazz, publicSet, out, ok, superClazz);
-    }
-
-    for (Class<?> iface : clazz.getInterfaces()) {
-      ok = checkSuperClass(clazz, publicSet, out, ok, iface);
-    }
-
     // TODO check generic type parameters
 
-    Field[] fields = clazz.getDeclaredFields();
+    Field[] fields = clazz.getFields();
```

Comment:
Added partial support for protected methods.  Thinking we can open an issue to cover protected methods in super classes.  Really need test for tricky cases like this.  I think it would be better to refactor the code to make it easier to test and then add support for protected in superclass with test.